accessibility/keyboard-accessibility/keyboard-accessibility.md

```diff
@@ -0,0 +1,64 @@
+# Keyboard-Only Navigation
+Sparkbox recognizes the importance of creating a web that is accessible to everyone, so we prioritize developing and testing for accessibility as an integral part of our process. In addition to automated testing, our developers conduct ongoing accessibility tests as they build their projects.  Manually testing content and custom site features for keyboard-only accessibility during this phase allow developers to identify and remediate issues before they become a problem for a user.
+
+To support this approach, we incorporate keyboard-only navigation testing into both our planning and development processes. Story cards and pull requests should include clear instructions for testing with a keyboard, to ensure that developers and testers who may not be familiar with web accessibility concepts can still successfully participate in creating accessible work.
+
+## Why Do We Value This?
+Keyboard accessibility determines the ability of a user to successfully navigate all areas and functionality of a website using only their keyboard.  Users who most commonly benefit from keyboard-only navigation are those with visual impairments and those with motor difficulties that make using a mouse difficult or impossible. Keyboard-only navigation is especially important as it serves as the basis for a wide range of assistive technologies, such as voice input and single switch entry devices. 
+
+## Developing and Testing for Keyboard Accessibility 
+Keyboard-only navigation tests will not reflect a real user’s experience unless system-wide keyboard accessibility is enabled. Instructions for enabling keyboard accessibility on different platforms are below.
+
+### Keyboard Accessibility on MacOS
+Keyboard accessibility is NOT enabled by default on a Mac. The default keyboard settings only allow the user to tab to form controls. System-wide keyboard accessibility must be enabled in both System Preferences and Safari Preferences to allow the user to be able to tab to other elements.
```

Comment:
I understand that the all caps “NOT” in this section is meant to be used as emphasis. I would recommend leaving this as lower case and using either the emphasis (*) or the strong emphasis (**) instead. A screen reader will come across this text and read it as N-O-T instead of the word.